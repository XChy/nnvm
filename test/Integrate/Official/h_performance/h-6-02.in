version https://git-lfs.github.com/spec/v1
oid sha256:184d06e9ec0ef832f5f8cff4a0f479a0d4bb6c551c56f4e502f416d573cf8634
size 10
