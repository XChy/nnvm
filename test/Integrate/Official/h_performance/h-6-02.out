0x1p+19
0